c/planarityCommandLine.c

```diff
@@ -416,15 +447,32 @@ int runGraphTransformationTest(char *command, char *infileName, int inputInMemFl
 
 	if (Result == OK)
 	{
+		// We need to capture whether output is 0- or 1-based to construct the name of the file to compare actualOutput with
+		int zeroBasedOutputFlag = 0;
 		char *actualOutput = NULL;
-		Result = TestGraphFunctionality(command, infileName, inputString, NULL, &actualOutput);
+		// We want to handle the test being run when we read from an input file or read from a string,
+		//	so pass both infileName and inputString.
+		// We want to output to string, so we pass in the address of the actualOutput string.
+		Result = TestGraphFunctionality(command, infileName, inputString, &zeroBasedOutputFlag, NULL, &actualOutput);
 		
 		if (Result != OK || actualOutput == NULL)
 		{
 			ErrorMessage("Failed to perform transformation to produce .g6 output.\n");
 		}
 		else
 		{
+			// Final arg is baseFlag, which is dependent on whether the FLAGS_ZEROBASEDIO is set in a graphP's internalFlags

```

Comment:
I think you have made an improvement such that this comment isn't needed anymore.